run_ci.sh

```diff
@@ -140,6 +140,9 @@ for PROJ in installer ; do
     mv /home/notstack/$PROJ $HOME/go/src/github.com/$GITHUB_ORGANIZATION
 done
 
+# If you wish the deployment to install a custom MAO then set the following to true
+export TEST_CUSTOM_MAO=false
```

Comment:
I think we need to default this somewhere else, or the variable will be undefined for existing users running the 11 script?

Maybe move this to common.sh?